### DISCUSSION TITLE
Marine Forecast that is "Parse able"

### DISCUSSION THREAD

**Discussion by PhilferroResideo** [weather-gov/api#775] [General]
Is there a NOAA Marine forecast source that has more fields broken out instead of one big lump of forecast text?
https://api.weather.gov/products/2965ad60-8ec7-45ce-b6f5-e7b7234f3f79 (New York)

**Comment by dwhitemv25** [comment]
This is a continuation from #774. 

MapClick supports alternate formats. Add CGI parameter `FcstType=dwml` for XML output or `FcstType=json` for a JSON representation.
